backend/src/test/java/de/trailmate/backend/repository/TourServiceTest.java

```diff
@@ -45,14 +47,26 @@ void isGetSingleTourResponseCorrectly() {
     @Test
     void isAddTourAddingCorrectly() {
 
-        Mockito.when(tourRepository.addTour(testItem))
+        Mockito.when(tourRepository.save(testItem))
                 .thenReturn((testItem));
 
         Tour actual = tourService.addTour(testItem);
 
         Assertions.assertThat(actual).isEqualTo(testItem);
 
     }
```

Comment:
This test would fail if the mutable object `testItem` behaves immutably as expected. (s. comments Tour-Model)